addon/components/pix-label-wrapped.js

```diff
@@ -2,10 +2,11 @@ import Component from '@glimmer/component';
 
 export default class PixLabelWrapped extends Component {
   get className() {
-    const classes = ['pix-label', 'pix-label--wrapped'];
+    const classes = ['pix-label', 'pix-label-wrapped'];
 
     if (this.args.inlineLabel) classes.push('pix-label--inline-label');
-    if (this.args.isDisabled) classes.push('pix-label--disabled');
+    if (this.args.isDisabled) classes.push('pix-label-wrapped--disabled');
+    if (this.args.variant === 'tile') classes.push('pix-label-wrapped--variant-tile');
```

Comment:
Pourquoi pas ça ? (+ évolutif)

```suggestion
    if (this.args.variant) classes.push(`pix-label-wrapped--variant-${this.args.variant}`);
```

Reply:
On en a discuté IRL, pour l'instant on laisse comme ça et lorsqu'un autre type de variant arrivera on pourra en rediscuter.
Au moins ici on précise que `tile` est le seul variant accepté.